hoot-rnd/src/test/cpp/hoot/rnd/language/translators/HootServicesTranslationInfoClientTest.cpp

```diff
@@ -35,9 +35,6 @@
 #include <hoot/core/TestUtils.h>
```

Comment:
After the move to HootNetworkRequest, the build request style tests needed to become more granular to just test the request data populating the request itself.